client/public/config/configuration.json

```diff
@@ -17,9 +17,6 @@
     "EXISTING_NR_TIMER_NAME": "existingNrTimer",
     "EXISTING_NR_TIMEOUT_MS": 0,
     "DEV_EXISTING_NR_TIMEOUT_MS": 0,
-    "PAYMENT_COMPLETION_TIMER_NAME": "paymentCompletionTimer",
-    "PAYMENT_COMPLETION_TIMEOUT_MS": 120000,
-    "DEV_PAYMENT_COMPLETION_TIMEOUT_MS": 900000,
```

Comment:
Excellent. Please add a note to the next Namerequest UI "release ticket" to delete these items from the config map (ie, when we deploy).

https://app.zenhub.com/workspaces/entity-5bf2f2164b5806bc2bf60531/issues/bcgov/entity/6065